src/servers/ZoneServer2016/entities/vehicle.ts

```diff
@@ -853,7 +865,8 @@ export class Vehicle2016 extends BaseLootableEntity {
   hasVehicleKey(server: ZoneServer2016): boolean {
     return (
       !!this.getItemById(Items.VEHICLE_KEY) ||
-      !!this.getDriver(server)?.getItemById(Items.VEHICLE_KEY)
+      !!this.getDriver(server)?.getItemById(Items.VEHICLE_KEY) ||
```

Comment:
do we still need to do this call ? since the driver is also a passenger and get checked after that again?

Reply:
I just kept it for redundancy... I don't believe it is needed though